src/main/java/ch/njol/skript/conditions/CondHasResourcePack.java

```diff
@@ -37,8 +37,7 @@
 public class CondHasResourcePack extends PropertyCondition<Player> {
 
 	static {
-		if (Skript.methodExists(Player.class, "hasResourcePack"))
-			register(CondHasResourcePack.class, PropertyType.HAVE, "[a] resource pack [(loaded|installed)]", "players");
+		register(CondHasResourcePack.class, PropertyType.HAVE, "[a] resource pack [(loaded|installed)]", "players");
```

Comment:
Doesn't this still require the check? I can't see the Player#hasResourcePack() in Spigot 1.13.2.

Reply:
Based on the fact `@RequiredPlugins` state "Paper 1.9 or newer" it is possible. As such I did go and check latest spigot releases and there's no method for `hasResourcePack` so this check should still remain

Reply:
yes this was only on the list for documentation update